content/docs/command-reference/init.md

```diff
@@ -126,9 +126,9 @@ include:
 - SCM other than Git is being used. Even though there are DVC features that
   require DVC to be run in the Git repo, DVC can work well with other version
   control systems. Since DVC relies on simple text
-  [DVC-files](/doc/user-guide/dvc-file-format) to manage <abbr>pipelines</abbr>,
-  data, etc, they can be added into any SCM thus providing large data files and
-  directories versioning.
+  [`.dvc` files](/doc/user-guide/dvc-file-format) to manage
```

Comment:
It's "simple `dvc.yaml` files" here. But linked, please.

Reply:
Hm. Wasn't totally fixed. Moved to https://github.com/iterative/dvc.org/pull/1372#discussion_r435676718 and https://github.com/iterative/dvc.org/pull/1372, which I'll commit.